modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/DataTypesPlannerTest.java

```diff
@@ -68,7 +68,8 @@ private void doTestSetOpNumbersCast(
         SqlTypeName[] numTypes = new SqlTypeName[] {SqlTypeName.TINYINT, SqlTypeName.SMALLINT, SqlTypeName.REAL,
             SqlTypeName.FLOAT, SqlTypeName.INTEGER, SqlTypeName.BIGINT, SqlTypeName.DOUBLE, SqlTypeName.DECIMAL};
 
-        boolean notNull = !nullable1 && !nullable2;
+        // TODO https://issues.apache.org/jira/browse/CALCITE-7062 : below.
+        // boolean notNull = !nullable1 && !nullable2;
```

Comment:
Let's do something like:
```
        // TODO https://issues.apache.org/jira/browse/CALCITE-7062 Change notNull1 and notNull2 to notNull after fix.
        //String notNull = !nullable1 && !nullable2 ? " NOT NULL" : "";
        String notNull1 = !nullable1 ? " NOT NULL" : "";
        String notNull2 = !nullable2 ? " NOT NULL" : "";
```
And  check it like:
```
                                : input(0, checkProject("TABLE1", "[CAST($t0):" + targetT + notNull1 + ", $t1]")))
...
                                : input(1, checkProject("TABLE2", "[CAST($t0):" + targetT + notNull2 + ", $t1]")))
```
To reduce commented code size and duplication